### PR TITLE
Fix #40. Add success callback

### LINKED ISSUE #40: Callback when tests are done.
Right now there's only an error callback in the JS API (runTests). It'd be nice to have a callback when all tests are done, regardless if it was successful or not. Or perhaps just add an additional success callback as well. This is especially useful when using the API together with e.g. gulp.

Thanks for a great job!

### PR DESCRIPTION
**Pull request by internettrans**
See #40. This could be considered a breaking change if anybody is depending on the old error callback not being called unless there is an error. I think I'll release this as 3.0.0.

cc @blittle @henebb

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ export function run(args) {
 	}
 
 	jsApi.runTests(config, function(err, safeExit, optionalMessage) {
-		console.log(chalk.red(err && err.stack ? err.stack : err));
+		if (err) {
```
Comment (internettrans) [inline src/cli.js:44]: This is now an errBack, so there isn't always an error

File: src/node-jspm-jasmine.js [hunk 2]
```diff
@@ -329,8 +329,7 @@ function importTestFiles(SystemJS, jasmine, specDir, specFiles, coverage, errCal
 				} else {
 					console.log(chalk.yellow(`Failed to generate coverage because of an error that occurred within remap-istanbul or istanbul. Not failing tests because this isn't your fault`));
 				}
-				timer.finish();
```
Comment (internettrans) [inline src/node-jspm-jasmine.js:332]: Now that we call the errBack always, that in turn calls timer.finish() for us.

File: src/node-jspm-jasmine.js [hunk 3]
```diff
@@ -410,11 +408,14 @@ function getJasmineConfig(config) {
...
 }
 
-function finishTestRun(passed, jasmine) {
+function finishTestRun(passed, jasmine, errCallback) {
 	notifyWatcherFinishedTestRun();
 	if (!isWatching()) {
 		const exitCode = passed ? 0 : 2;
 
+		const successResult = null;
+		errCallback(successResult);
```
Comment (internettrans) [inline src/node-jspm-jasmine.js:417]: Call the errBack with a success

**Review by blittle — APPROVED** [review]
Looks good to me

**Pull request merged**